essentials-framework/Essentials Core/PepperDashEssentialsBase/Global/Global.cs

```diff
@@ -107,7 +107,7 @@ public static bool IsRunningMinimumVersionOrHigher(string minimumVersion)
             // Check for beta build version
             if (runtimeVersionMajor == 0)
```

Comment:
Should this check for the additional keywords in the informational version? There won't be any version numbers with 0 in them anymore...